domains/certificates/Query.API/API/MasterDataService/data.json

```diff
@@ -8,6 +8,6 @@
             "techCode": "T010000"
         },
         "meteringPointOwner": "280d0d0e-acde-4132-9916-c2299f583325",
-        "meteringPointOnboarded": true
+        "meteringPointOnboarded": "2022-09-11T19:13:50+00:00"
```

Comment:
Is is correct, to move from a boolean to datetime?
If so, would it make sense to rename the field?

Reply:
Yes we need it to know when to start fetching data. This might be changed when datasync is properly implemented by heimdal.
renamed in 828ac08

Reply:
It is desired to have it as a timestamp. This allows us to say that certificates should be generated for a metering point starting at a specific timestamp - and the timestamp can be in the future (e.g. Jan 1st 2023)

I agree that it makes sense to rename the property. I suggest "issuanceStartDate"